opencensus-shim/src/test/java/io/opentelemetry/opencensusshim/OpenTelemetryB3FormatImplTest.java

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.opencensusshim;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import io.opencensus.trace.SpanContext;
+import io.opencensus.trace.SpanId;
+import io.opencensus.trace.TraceId;
+import io.opencensus.trace.TraceOptions;
+import io.opencensus.trace.Tracestate;
+import io.opencensus.trace.propagation.BinaryFormat;
+import io.opencensus.trace.propagation.SpanContextParseException;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Tests for {@link OpenTelemetryBinaryFormatImpl}. */
+@RunWith(JUnit4.class)
+public class OpenTelemetryB3FormatImplTest {
```

Comment:
By making a generic `OpenTelemetryTextFormat` we won't need all this testing, we'd be able to test a mock `TextMapPropagator`

Reply:
Oh wait - this is the binary format test XD. Rename the class. I guess we need tests for the textmappropagator wrapper too.